template.php

```diff
@@ -372,12 +372,22 @@ function campaignion_foundation_form_alter(&$form, $form_state, $form_id) {
  * JavaScripts to pick up.
  */
 function campaignion_foundation_form_webform_client_form_alter(&$form, &$form_state, $form_id) {
-  $settings['campaignion_foundation']['webform'] = [
-    'total_steps' => $form_state['webform']['page_count'],
-    'current_step' => $form_state['webform']['page_num'],
-    'last_completed_step' => $form_state['webform']['page_visited'] ?? 0,
-  ];
-  $form['#attached']['js'][] = ['data' => $settings, 'type' => 'setting'];
+  // Check if the current node id is the node id the webform was attached to.
+  // This is needed to deal with the situation when there are multiple webforms
+  // on a single node.
```

Comment:
For me the second paragraph would be enough explanation. I don’t think there is a need to explain what the if-condition does.